meetup/migrations/0001_initial.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.2 on 2016-03-04 15:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import meetup.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('pizzaplace', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Meetup',
+            fields=[
```

Comment:
Time stamps are pretty much always useful metadata...there's no reason not to include them from the start.